src/styles/main.scss

```diff
@@ -0,0 +1,115 @@
+@import './variables';
+@import './mixins';
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 0.5s;
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    @include font-fam('Roboto, sans-serif;', 12px, 500, 18px);
+
+    margin: 0 0 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    @include font-fam('Roboto, sans-serif;', 10px, 400, 14px);
+
+    margin: 0 0 16px;
+    color: $secondary-text-color;
+  }
+
+  &__button {
+    @include font-fam('Roboto, sans-serif;', 14px, 700, 16px);
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    background-color: $blue-color;
+    border-radius: 5px;
+    color: $white-color;
+  }
+
+  &__button:hover {
+    background-color: $white-color;
+    color: $blue-color;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  .price-text {
+    @include font-fam('Roboto, sans-serif;', 12px, 400, 18px);
```

Comment:
There is a semicolon inside the font family string in the mixin call. It should be removed to avoid syntax errors. The correct format should be without the semicolon inside the quotes.